components/molecule/avatar/src/index.js

```diff
@@ -5,7 +5,11 @@ import AtomImage from '@s-ui/react-atom-image'
 import AtomSkeleton from '@s-ui/react-atom-skeleton'
 import useConvertStringToHex from './useConvertStringToHex'
 import AvatarFallback from './AvatarFallback'
-import AvatarBadge from './AvatarBadge'
+import AvatarBadge, {
+  AVATAR_BADGE_STATUSES,
+  AVATAR_BADGE_PLACEMENTS,
+  AVATAR_BADGE_SIZES
+} from './AvatarBadge'
 
 export const AVATAR_SIZES = {
```

Comment:
rename it to `MOLECULE_AVATAR_STATUSES` plzz

Reply:
preserve the naming 4 retrocompatibility

Reply:
Ok, I will export like `MOLECULE_AVATAR_SIZES`

Reply:
@andresin87 https://github.com/SUI-Components/sui-components/pull/1359/commits/dfb8ad8e8bace0fbb4e9dd0f3c809e5248ecad9f